src/ui/pages/Overview/Overview.tsx

```diff
@@ -297,6 +354,50 @@ function OverviewComponent() {
     </CenteredFillViewportView>
   );
 
+  const { preferences, setPreferences } = usePreferences();
+  const isTestnetMode = Boolean(preferences?.testnetMode?.on);
+  useEffect(() => {
+    // reset filter chain when switching between modes
+    // so that we do not show unsupported network data
+    setFilterChain(null);
+  }, [isTestnetMode]);
+  const testnetGuardView = (
+    <CenteredFillViewportView
+      adjustForNavigationBar={true}
+      maxHeight={getMinTabContentHeight(offsetValuesState)}
+    >
+      <FillView>
+        <EmptyView2
+          title="Wrong Environment"
+          message={
+            <div>
+              {preferences?.testnetMode?.on ? (
+                <UnstyledButton
+                  className="underline hover:no-underline"
+                  onClick={() => {
+                    setPreferences({ testnetMode: null });
```

Comment:
should we make it `{ on: false }` in this case?

Reply:
My reasoning here is that this button is helpful to those who do not need testmode and ended up in it accidentally. Those who need test toggle to remain can use either the `T` shortcut or the toggle at the bottom of the page